docs/how-certificate-issuing-works.md

```diff
@@ -32,9 +32,9 @@ OIDC ID token.
 
 To authorize the token Fulcio must:
 
-- Use the issuer claim from the token to find the issuers OIDC discovery
+- Use the issuer claim from the token to find the issuer's OIDC discovery
```

Comment:
nit: This doesn't happen with each request, they're cached and fetched periodically.